xml/security_ad_support.xml

```diff
@@ -1303,4 +1303,121 @@
    </step>
   </procedure>
  </sect1>
+
+ <sect1 xml:id="sec-active-directory-certificate-auto-enrollment">
+  <title>&ad; certificate auto-enrollment</title>
+  <para>
+   Certificate auto-enrollment enables network devices to automatically
+   enroll certificates from Active Directory Certificate Services, including
+   &sles; devices, with no user intervention. This is managed by &ad;'s
+   Group Policy, using Samba's <command>samba-gpupdate</command> command.
+  </para>
+  <sect2 xml:id="sec-active-directory-auto-enrollment-server-configuration">
+   <title>Configuring certificate auto-enrollment on the server</title>
+   <para>
+    The Windows server roles "Certification Authority", "Certificate
+    Enrollment Policy Web Service", "Certificate Enrollment Web Service",
+    "Certification Authority Web Enrollment", and "Network Device Enrollment
+    Service" all must be installed and configured on the &ad; server.
+   </para>
+   <para>
+    Configure Group Policy auto-enrollment as described in this Microsoft
+    documentation: <link xlink:href="https://docs.microsoft.com/en-us/windows-server/networking/core-network-guide/cncg/server-certs/configure-server-certificate-autoenrollment#configure-server-certificate-auto-enrollment"/>.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec-active-directory-auto-enrollment-client-configuration">
+   <title>Enable certificate auto-enrollment on the client</title>
+   <para>
+    Follow the steps in the following procedure to enable certificate on
+    your clients.
+   </para>
+   <procedure xml:id="pro-enable-certification-auto-enrollment-client">
+    <step>
+    <para>
+     Install the <package>samba-gpupdate</package> package. This will
+     automatically install the <package>certmonger</package>,
+     <package>cepces</package>, and <package>sscep</package> dependencies.
+     Samba uses <command>sscep</command> to download the Certificate
+     Authority root chain, then uses the
+     <systemitem class="daemon">certmonger</systemitem>
+     paired with <systemitem>cepces</systemitem> to monitor the host
+     certificate templates.
+     </para>
+    </step>
+    <step>
+     <para>
+      Join to an Active Directory domain (one where the CA has been
+      previously configured as explained in <xref linkend="sec-active-directory-auto-enrollment-server-configuration"/>).
+     </para>
+    </step>
+    <step>
+     <para>
+      On Winbind-joined machines, set the <filename>smb.conf</filename>
+      global parameter by adding the line
+      <literal>apply group policies = yes</literal>.
+     </para>
+    </step>
+    <step>
+    <para>
+     For SSSD-joined machines, install <package>samba-gpupdate</package>
```

Comment:
You have a typo here. The package name is oddjob-gpupdate, not samba-gpupdate (this is a different package).

Reply:
This is not resolved. Why did you merge before fixing the incorrect package name here, as I pointed out?